sdk/python/feast/pyspark/abc.py

```diff
@@ -334,3 +352,18 @@ def offline_to_online_ingestion(
             IngestionJob: wrapper around remote job that can be used to check when job completed.
         """
         raise NotImplementedError
+
+    @abc.abstractmethod
```

Comment:
Since Dataproc Launcher and Standalone Launcher hasn't implemented this change, this currently breaks the python linter CI stage and also breaks Dataproc and Standalone Launcher functionalities since they can't be instantiated.

Perhaps we can create the methods for the two Launchers, and raise NotImplementedError for the time being? Then we can either add the implementation directly via this PR (either me or @pyalex), or with a follow up PR?